### ISSUE TITLE
Attestation fetching stopped

### ISSUE THREAD

**Issue by bonomat** [itchysats/itchysats#2002]
According to our logs it looks like we stopped fetching attestations between 2022-05-10T08:01:13.144388844Z and 2022-05-10T22:50:21.301540243Z(after a restart).

**Comment by bonomat** [comment]
Appearances of the log message `Fetching attestation` per hour over the last 2 days: 

![image](https://user-images.githubusercontent.com/224613/167741401-c821ed26-49f4-4eac-ad45-067478eb9fba.png)

**Comment by bonomat** [comment]
# Analysis

My interpretation of this issue is that this is not an issue: 

- Each DLC has a `settlement_event_id` which is the event id for when this CFD will expire unless it is rolled over
- During startup we load all CFDs from the DB and add their `settlement_event_id` to a list called `pending_attestation`: 

https://github.com/itchysats/itchysats/blob/b29277675c5fbb89c3f3c24369de35b62b404e4e/daemon/src/oracle.rs#L308-L342

https://github.com/itchysats/itchysats/blob/b29277675c5fbb89c3f3c24369de35b62b404e4e/daemon/src/oracle.rs#L234-L240

- on a regular interval we check if this attestation has _likely_ occurred yet and will try to fetch it from the oracle: 

https://github.com/itchysats/itchysats/blob/b29277675c5fbb89c3f3c24369de35b62b404e4e/daemon/src/oracle.rs#L185-L189

- on every rollover or contract completion, we also add an event id into this list, by sending `MonitorAttestation` to the oracle actor, e.g. here: 

https://github.com/itchysats/itchysats/blob/d6f8c9955631c18f2a31920c39de350049e486a7/daemon/src/process_manager.rs#L95-L99

and here: 

https://github.com/itchysats/itchysats/blob/d6f8c9955631c18f2a31920c39de350049e486a7/daemon/src/process_manager.rs#L198-L202


# Conclusion
- This means that when starting up and with no down time, we likely get a list of events to monitor 24h from now
- On every rollover we add a new id to this list, so that it can be fetched in the future, i.e. one in 24+1h from now
- Because our maker (testnet and production) gets regularly restarted, this list mostly contains events for 24h in the future and none for the next hour
- This means, we won't be fetching any attestation for the next 24h. 
- However, if our application runs for over 24h, we expect to be fetching for new attestations every 1h. 

Long story short, this does not seem to be an issue.
